traits.html

```diff
@@ -1,29 +1,35 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <meta charset="UTF-8">
   <title>Traits</title>
   <link rel="stylesheet" href="traits.css">
 </head>
 
-<body>
+<center>
+  <body>
+    <section class="traitsContainer" id="traitsContainer">
+      <div class="" id="">
+        <h2>Here are the results</h2>
```

Comment:
We already called the immediate feedback 'results'. Maybe change this to "Report of all traits"?

Reply:
has this been addressed?

Reply:
can someone answer this question ?